src/PKSim.CLI.Core/Services/SimulationExporter.cs

```diff
@@ -206,12 +205,5 @@ private async Task exportIndividualSimulationResultsToExcelAsync(IndividualSimul
          await _simulationExportTask.ExportResultsToExcelAsync(simulation, simulationResultFileFullPath, launchExcel:false);
          _logger.AddDebug($"Exporting simulation results to '{simulationResultFileFullPath}'", simulationExportOptions.LogCategory);
       }
-
-      private void exportParameters(IndividualSimulation individualSimulation, SimulationExportOptions simulationExportOptions)
```

Comment:
We never used that. It's left from early on and does notb ring anuthing. Removing